cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxEditBoxHelper.java

```diff
@@ -172,6 +182,7 @@ public void run() {
                             Log.d(TAG, "edit box get focus");
                         } else {
                             editBox.setVisibility(View.GONE);
+                            mHandler.postDelayed(decor_view_settings, 500);
```

Comment:
there is another ` mCocos2dxActivity.hideVirtualButton();` call in the following statements. Should we remove that one and make it called delayed across all the Android devices?